packages/fxa-content-server/app/scripts/views/settings/clients.js

```diff
@@ -155,6 +154,10 @@ const View = FormView.extend(
           }
         }
 
+        if (item.scope) {
+          item.title = `${item.title} - ${item.scope}`;
```

Comment:
There was at least one complaint about adding the scopes to the title, but I can't for the life of me find it.

Reply:
That honestly makes sense, I did think this was a little weird (I didn't add this, I just moved it below the if/elses because I alter the title `if item.deviceId === null`):

<img width="532" alt="image" src="https://user-images.githubusercontent.com/13018240/70066847-ff4fc000-15b2-11ea-8cd8-d4d0263f69f2.png">

We don't appear to be using `scope` anywhere else. Maybe we don't need it?